code/datums/emergency_calls/pred_hunt/hunting_calls.dm

```diff
@@ -248,20 +250,55 @@
 	if(SSticker.mode)
 		SSticker.mode.initialize_predator(hunter, ignore_pred_num = TRUE)
 
-/datum/emergency_call/young_bloods/inexperienced
+/datum/emergency_call/young_bloods/one_member //For if a pred wants to teach a single youngblood
+	name = "Hunting Grounds - Solo Youngblood"
+	blooding_name = "Solo Youngblood (One member)"
+	mob_max = 1
+	mob_min = 1
+	time_required_for_job = 5 HOURS
+	youngblood_time = 0 HOURS
+	youngblood_time_required_for_job = 0 HOURS
+
+/datum/emergency_call/young_bloods/one_member/experienced //For if a pred wants to teach a more experienced youngblood but still one on one
+	name = "Hunting Grounds - Solo Youngblood (Experienced)"
+	blooding_name = "Solo Youngblood (One member - Experienced)"
+	youngblood_time = 7 HOURS
+	youngblood_time_required_for_job = 5 HOURS
+
+/datum/emergency_call/young_bloods/three_members
 	name = "Hunting Grounds - Inexperienced Youngblood Party" //For completly new youngblood players
 	blooding_name = "Inexperienced Youngblood Party (Three members)"
+	mob_max = 3
+	mob_min = 2
```

Comment:
if the max is 3 and the min is two theres going to be times when theres two people

Reply:
![illustration-happy-cartoon-star_546897-1371](https://github.com/user-attachments/assets/c9f9f896-8c94-488c-ad8e-779bcc94d2a6)